packages/salesforcedx-utils-vscode/src/telemetry/configUtil.ts

```diff
@@ -44,8 +56,8 @@ export class ConfigUtil {
         const rootPath = getRootWorkspacePath();
         const myLocalConfig = await ConfigFile.create({
           isGlobal: false,
-          rootFolder: path.join(rootPath, '.sf'),
-          filename: 'config.json'
+          rootFolder: path.join(rootPath, Global.SFDX_STATE_FOLDER),
+          filename: 'sfdx-config.json'
```

Comment:
should this value be in packages/salesforcedx-sobjects-faux-generator/src/constants.ts?